ClientApp/src/models/Ticket.ts

```diff
@@ -0,0 +1,10 @@
+export default interface Ticket {
+    id: string;
+    name: string;
+    description: string;
+    ticketType: string;
+    ticketStatus: string;
+    reporterId?: string;
+    assigneeId?: string;
+    parentTicketId?: string;
```

Comment:
So like you pointed out, we should have due date, tags, and priority as well?

Reply:
Feature for beta.